model/state_model.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy import *
+from .base import Base
+from sqlalchemy.orm import relationship
+
+
+class Country(Base):
```

Comment:
you called the filename `state_model.py` but it also has country model, you should separate each model into it's own file